Vagrantfile

```diff
@@ -31,6 +31,28 @@ Vagrant.configure("2") do |config|
     fedora.vm.synced_folder ".", "/vagrant", type: "rsync"
   end
 
+  config.vm.define "rawhide" do |rawhide|
+    def get_box(pattern)
+      require 'open-uri'
+      require 'nokogiri'
+
+      url = "https://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Cloud/x86_64/images/"
+      doc = Nokogiri::HTML(URI.open(url))
+      box = doc.css('a').map { |link| link['href'] }.select { |alink| alink.include?(pattern) }.last
+      url + box
```

Comment:
I think this could lead to a problem with the cirrus-ci cache.
Currently we're caching the result of the Vagrant image building to speed up tests and safe cycles. The "key" to the cache item is the `VagrantFile`.

Maybe we could add this as another cirrus job?

Reply:
That way the we could determine the rawhide image on the host machine, use it in combination with the VagrantFile as cache key, log it (!), and pass it to vagrant via env var...

Reply:
I think everything got addressed by splitting the caches in a per ${DISTRO} cache.
Apparently, the previous cache did not include all the images.
This should also protect any potential size limit (did not find any reference in the doc but I read there's a 2GB hard limit in some issues).
For rawhide, which might need to be updated more frequently, the local cache gets invalidated once a week.